code/client/src/components/ERC20Grid.jsx

```diff
@@ -205,12 +199,11 @@ export const ERC20Grid = ({ address }) => {
           console.error(ex)
         }
       }))
-      // console.log('tts merged', tts)
       if (cancelled) {
         return
       }
-      // tts = uniqBy(tts, t => t.key)
-      setCurrentTrackedTokens(tts)
+      // Merge to existing list.
+      setCurrentTrackedTokens(ptts => uniqBy(t => t.key, [...ptts, ...tts]))
```

Comment:
should still filter out `untrackedTokenKeys` and merge with `defaultTrackedTokens`

Reply:
done